example/app.js

```diff
@@ -35,6 +36,18 @@ class App extends React.Component {
         description: 'Custom class for slides & thumbnails'
       },
       {
+        original: `${PREFIX_URL}2.jpg`,
+        thumbnail: `${PREFIX_URL}2t.jpg`,
+        description: 'Custom bullets with custom actions',
+        bulletClass: 'red-bullet',
```

Comment:
Lets remove this from the example, it was a bit confusing to me when I first saw the slide with a red dot.